### PR TITLE
Update fleet page

### PR DESCRIPTION
**Pull request by npentrel**

### CODE REVIEW

File: docs/manage/fleet/_index.md [hunk 1]
```diff
@@ -23,23 +23,20 @@ All communication happens securely over HTTPS using secret tokens that are in a
...
 ## Locations
 
-{{< readfile "/static/include/manage/locations.md" >}}
+In Viam, every robot belongs to a location.
+A location is a virtual grouping of robots that allows you to organize robots and manage access.
 
 ![An image of two locations, New York, and Chicago, in one organization, Good Robots](../img/locations.png)
 
-For information on how to manage locations, see [Locations](locations).
+For information on location and how to manage locations, see [Locations](locations).
```
Comment (JessamyT) [inline docs/manage/fleet/_index.md:35]:
```suggestion
For information on locations and how to manage them, see [Locations](locations).
```

File: docs/manage/fleet/_index.md [hunk 2]
```diff
@@ -23,23 +23,20 @@ All communication happens securely over HTTPS using secret tokens that are in a
...
 
 ## Organization
 
-{{< readfile "/static/include/manage/organizations.md" >}}
+An organization is a group of one or more locations that helps you organize your fleet and manage access.
 
 {{<gif webm_src="../img/organizations.webm" mp4_src="../img/organizations.mp4" alt="An organization for personal robots and one for work robots.">}}
```
Comment (JessamyT) [inline docs/manage/fleet/_index.md:41]: Out of diff but this is such a cute GIF-- could it be made larger and also higher resolution?

**Review by JessamyT — APPROVED** [review]
LGTM % one suggestion

File: docs/manage/fleet/_index.md (hunk 2 above)
Comment (npentrel) [reply docs/manage/fleet/_index.md:41]: That would be a cristina ask - so maybe in the future

**Pull request merged**